core/unix/signal_linux_x86.c

```diff
@@ -519,13 +521,19 @@ sigcontext_to_mcontext_simd(priv_mcontext_t *mc, sig_full_cxt_t *sc_full)
         if (ZMM_ENABLED()) {
             kernel_xstate_t *xstate = (kernel_xstate_t *)sc->fpstate;
             if (sc->fpstate->sw_reserved.magic1 == FP_XSTATE_MAGIC1) {
+                /* The following three XCR0 bits should have been checked already
+                 * ZMM_ENABLED().
```

Comment:
grammar nit: "checked already ZMM_ENABLED" is missing a preposition: "by" or "in" or some such.  Ditto below.

Reply:
Fixed.